src/wp-admin/includes/deprecated.php

```diff
@@ -703,7 +703,7 @@ function get_others_unpublished_posts( $user_id, $type = 'any' ) {
 	else
 		$type_sql = " ( post_status = 'draft' OR post_status = 'pending' ) ";
 
-	$dir = ( 'pending' == $type ) ? 'ASC' : 'DESC';
+	$dir = ( 'pending' === $type ) ? 'ASC' : 'DESC';
```

Comment:
```suggestion
	$dir = 'pending' === $type ? 'ASC' : 'DESC';
```

Reply:
Please keep the parentheses. Operator precedence can cause havoc, so preventing issues with that by using parentheses is always a good thing.

Reply:
Please keep the parentheses. Operator precedence can cause havoc, so preventing issues with that by using parentheses is always a good thing.

Reply:
Please keep the parentheses. Operator precedence can cause havoc, so preventing issues with that by using parentheses is always a good thing.